### ISSUE TITLE
[WebUI] Reddit video generation 

### ISSUE THREAD

**Issue by MatteoFasulo** [MatteoFasulo/Whisper-TikTok#31]
Adapt the work of @duozokker into the WebUI ("reddit" page) allowing the user to modify the json file (as for main video.json file)

**Expected behavior**
User should be able to select which video to generate and modify the json file directly from the WebUI

**Screenshots**
WebUI

**Additional context**
Additional features will be a transition with "follow" pop-up animation, reddit (submission) title on screen with post box template containing the submission content

**Comment by YotomiY** [comment]
Hello!
How is it going?

**Comment by YotomiY** [comment]
When is this coming?

**Comment by MatteoFasulo** [comment]
> When is this coming?

Hi, I’m currently dedicating a few hours each week to this project, so I’m unable to provide a specific completion date at this time. I appreciate your understanding.

**Comment by YotomiY** [comment]
Hello,
Im sorry to ask this but is this project dead

**Comment by MatteoFasulo** [comment]
> Hello, Im sorry to ask this but is this project dead

Not really, did not have enough time to push all the features I wanted to implement. Right now I am working more deeply into Whisper model rather than this tool for creating videos. Planning to do a huge refactoring of the code making it more modular as to let users add their components into the project independently; also looking for other contributors to join the project and push it forward :)
